src/main/java/uk/co/tfn/HelperMethods.java

```diff
@@ -360,4 +358,105 @@ public void sendKeysById(String id, String input) {
+                // 1. Max age, no min age
+                this.sendKeysById("age-range-max", "30");
+                break;
+            case 2:
+                // 2. Min age, no max age
+                this.sendKeysById("age-range-min", "12");
+                break;
+            case 3:
+                // 3. Max and min age, diff values
+                this.sendKeysById("age-range-min", "13");
+                this.sendKeysById("age-range-max", "18");
+                break;
+            case 4:
+                // 4. Max and min age, same values
+                this.sendKeysById("age-range-min", "50");
+                this.sendKeysById("age-range-max", "50");
+                break;
+        }
+    }
+
+    public void completeUserDetailsPage() {
+        int randomSelector = randomNumberBetweenOneAnd(4);
+        switch (randomSelector) {
+            case 1:
+                // 1. No to both questions
+                this.clickElementById("age-range-not-required");
+                this.clickElementById("proof-not-required");
+                this.continueButtonClick();
+                break;
+            case 2:
+                // 2. No to age limit, Yes to Proof
+                this.clickElementById("age-range-not-required");
+                this.clickElementById("proof-required");
+                this.randomlyChooseAProof();
+                this.continueButtonClick();
+                break;
+            case 3:
+                // 3. Yes to age limit, Yes to Proof
+                this.clickElementById("age-range-required");
+                this.randomlyChooseAgeLimits();
+                this.clickElementById("proof-required");
+                this.randomlyChooseAProof();
+                this.continueButtonClick();
+                break;
+            case 4:
+                // 4. Yes to age limit, No to Proof
+                this.clickElementById("age-range-required");
+                this.randomlyChooseAgeLimits();
+                this.clickElementById("proof-not-required");
+                this.continueButtonClick();
+        }
+    }
+
+    public void randomlyDetermineUserType() {
+        int randomSelector = randomNumberBetweenOneAnd(2);
+        switch (randomSelector) {
+            case 1:
+                // 1. Click Any and continue
+                this.clickElementById("passenger-type0");
+                this.continueButtonClick();
+                this.waitForPageToLoad();
+                break;
+            case 2:
+                // 2. Click a non-Any, complete the next page, and continue
+                int randomUserType = randomNumberBetweenOneAnd(6);
+                this.clickElementById(String.format("passenger-type%s", String.valueOf(randomUserType)));
+                this.continueButtonClick();
+                this.waitForPageToLoad();
+                this.completeUserDetailsPage();
+                break;
+        }
+    }
+
+    public static int randomNumberBetweenOneAnd(int x) {
```

Comment:
Maybe move this above the constructor, just so it's with the other static methods